### PR TITLE
Added HyperLogLog support to Redis client

### PR DESCRIPTION
**Pull request by SSPkrolik**
Implemented 3 additional commands which implement the whole set of commands to work with Redis HyperLogLog data type:

1) pfadd
2) pfcount
3) pfmerge

### CODE REVIEW

File: lib/pure/redis.nim [hunk 1]
```diff
@@ -798,6 +798,35 @@ proc zunionstore*(r: Redis, destination: string, numkeys: string,
   
   return r.readInteger()
 
+# HyperLogLog
+
+proc pfadd*(r: Redis, key: string, elements: varargs[string]): RedisInteger = 
+  ## Add variable number of elements into special 'HyperLogLog' set type
+  var args = @[key]
```
Comment (Araq) [inline lib/pure/redis.nim]: Use 'let' instead of 'var' here (if possible).

File: lib/pure/redis.nim [hunk 2]
```diff
@@ -798,6 +798,35 @@ proc zunionstore*(r: Redis, destination: string, numkeys: string,
...
+
+  return r.readInteger()
+
+proc pfmerge*(r: Redis, destination: string, sources: varargs[string]) =
+  ## Merge several source HyperLogLog's into one specified by destKey
+  var args = @[destination]
+  for source in items(sources):
+    args.add(source)
+
+  r.sendCommand("PFMERGE", args)
```
Comment (Araq) [inline lib/pure/redis.nim]: Create a version of 'sendCommand' that can take a single string? Saves some seq creations.

**Comment by SSPkrolik** [comment]
@Araq, made it way more efficient by eliminitating sequences creation code and using another variant of the 'sendCommand' that was already implemented.

**Comment by Araq** [comment]
@dom96 please review.

**Comment by dom96** [comment]
Looks good to me.

**Pull request merged**